.github/workflows/deploy.yml

```diff
@@ -42,5 +42,7 @@ jobs:
         run: vercel deploy --prebuilt --prod --token=${{ secrets.VERCEL_TOKEN }}
       # Trigger Algolia crawler
       - name: Crawl
+        uses: algolia/algoliasearch-crawler-github-actions@v1
```

Comment:
Is this really needed when we use the REST API?

Reply:
haha, no. Seems like I just forgot it after trying it out 😅 
```suggestion
```